internal/githubprovider/model.go

```diff
@@ -139,11 +142,28 @@ func fromIssue(batch *dvmodel.Batch, input *github.Issue) error {
 			return fmt.Errorf("unsupported pmbodyparser.Kind: %v", relationship.Kind)
 		}
 	}
-
 	batch.Tasks = append(batch.Tasks, &issue)
 	return nil
 }
 
+func parseDuration(body string) string {
+	rules := regexp.MustCompile(`\b\w+\b`)
+	words := rules.FindAllString(body, -1)
+	previous := ""
+	for _, v := range words {
+		if previous == "time" {
```

Comment:
Regexp for splitting word is not efficient.

Please use full regexp that returns the target in a match instead of mixing regexp for parsing words and manual iterations.

Something like `time\ (…)`